mgl-webapp/src/main/java/com/organization/mvcproject/service/GameServiceImpl.java

```diff
@@ -1,68 +1,43 @@
 package com.organization.mvcproject.service;
 
-import java.util.ArrayList;
+
 import java.util.List;
 
 import org.springframework.stereotype.Service;
 
+import com.organization.mvcproject.dao.MockDAO;
 import com.organization.mvcproject.model.Game;
 
 
 
 @Service
 public class GameServiceImpl implements GameService {
 
-	/**
-	 * TODO 2.0 The class that interacts with persistent data is called a Data Access Object(DAO)
-	 *  or a Repository class. The private static list is mocking our persistance of games.
-	 *   
-	 *  Move this list, and methods operating on this list to an appropriately named package and class.
-	 */
+	private MockDAO gameDAO; 
+	
+	public GameServiceImpl() {
+		gameDAO = new MockDAO();
```

Comment:
Use Spring to handle your dependency injection. This will work, but if you're not letting Spring handle any dependencies in MockDAO by explicitly declaring it in memory here.